### PR TITLE
Node file block clean up

### PR DESCRIPTION
**Pull request by Caleb-Hill**

### CODE REVIEW

File: splinterd/src/node_id.rs [hunk 1]
```diff
@@ -0,0 +1,64 @@
+use rand::Rng;
```
Comment (agunde406) [inline splinterd/src/node_id.rs:1]: Missing copyright

File: splinterd/src/daemon.rs [hunk 2]
```diff
@@ -246,10 +247,17 @@ impl SplinterDaemon {
         let admin_service_verifier = secp256k1_context.new_verifier();
         let auth_config_verifier = secp256k1_context.new_verifier();
         let signing_context = Arc::new(Mutex::new(secp256k1_context));
+        let node_id: String = crate::node_id::get_node_id(
```
Comment (agunde406) [inline splinterd/src/daemon.rs:250]: Add a use statement above for get_node_id instead of useing the whole crate::node_id::get_node_id

**Review by agunde406 — CHANGES_REQUESTED** [review]

File: splinterd/src/daemon.rs [hunk 3]
```diff
@@ -1272,9 +1283,12 @@ impl SplinterDaemonBuilder {
             CreateError::MissingRequiredField("Missing field: heartbeat".to_string())
         })?;
 
-        let node_id = self.node_id.ok_or_else(|| {
-            CreateError::MissingRequiredField("Missing field: node_id".to_string())
-        })?;
+        let node_id = self
```
Comment (agunde406) [inline splinterd/src/daemon.rs:1286]: This should remain an option here and not be generated if None, that happens in your get_node_id() function that is called in SplinterDaemon.start(). Not this means that it should not be returning an error here.

File: splinterd/src/daemon.rs [hunk 4]
```diff
@@ -1318,9 +1318,12 @@ impl SplinterDaemonBuilder {
             CreateError::MissingRequiredField("Missing field: initial_peers".to_string())
         })?;
 
-        let display_name = self.display_name.ok_or_else(|| {
-            CreateError::MissingRequiredField("Missing field: display_name".to_string())
-        })?;
+        let display_name = self
```
Comment (agunde406) [inline splinterd/src/daemon.rs:1321]: This should also remain an option and use the node id that is created in SplinterDaemon.start() if display name is not set.

**Review by agunde406 — CHANGES_REQUESTED** [review]

**Review by agunde406 — APPROVED** [review]
Tested

**Review by peterschwarz — APPROVED** [review]

**Pull request merged**